logstash-core/spec/logstash/config/source/local_spec.rb

```diff
@@ -0,0 +1,410 @@
+# encoding: utf-8
```

Comment:
This files looks scarier than it really is, but I wanted to make sure we really cover all the case we had before.
By looking at our code and the tests, I've found out that we were missing a few of them.